applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -423,52 +466,253 @@ static void le_audio_msg_sub_thread(void)
 			}
 
 			LOG_INF("Presentation delay %d us is set by initiator", pres_delay_us);
+
+			break;
+
+		case LE_AUDIO_EVT_PA_SYNC_LOST:
+			LOG_DBG("PA sync lost");
+
+			if (IS_ENABLED(CONFIG_BT_OBSERVER)) {
+				ret = bt_mgmt_scan_start(0, 0, BT_MGMT_SCAN_TYPE_BROADCAST, NULL);
+				if (ret) {
+					LOG_ERR("Failed to restart scanning: %d", ret);
+					break;
+				}
+
+				LOG_INF("Restarted scanning for broadcaster");
+			}
+
+			break;
+
+		case LE_AUDIO_EVT_NO_VALID_CFG:
+			LOG_WRN("No valid configurations found, will disconnect");
+
+			bt_mgmt_conn_disconnect(msg.conn, BT_HCI_ERR_REMOTE_USER_TERM_CONN);
+
 			break;
 
 		default:
-			LOG_WRN("Unexpected/unhandled event: %d", event);
+			LOG_WRN("Unexpected/unhandled le_audio event: %d", event);
+
 			break;
 		}
 
 		STACK_USAGE_PRINT("le_audio_msg_thread", &le_audio_msg_sub_thread_data);
 	}
 }
 
+/* Handle content control events */
+static void content_control_msg_sub_thread(void)
+{
+	int ret;
+
+	const struct zbus_channel *chan;
+
+	while (1) {
+		ret = zbus_sub_wait(&content_control_evt_sub, &chan, K_FOREVER);
+		ERR_CHK(ret);
+
+		struct content_control_msg msg;
+
+		ret = zbus_chan_read(chan, &msg, ZBUS_READ_TIMEOUT_MS);
+		ERR_CHK(ret);
+
+		uint8_t event = msg.event;
+
+		LOG_DBG("Received event = %d, current state = %d", event, strm_state);
+
+		switch (event) {
+		case MEDIA_PLAY:
+			ret = le_audio_play();
+			if (ret) {
+				LOG_ERR("Failed to play: %d", ret);
+			}
+
+			break;
+
+		case MEDIA_PAUSE:
+			ret = le_audio_pause();
+			if (ret) {
+				LOG_ERR("Failed to pause: %d", ret);
+			}
+
+			break;
+
+		default:
+			LOG_WRN("Unexpected/unhandled event: %d", event);
```

Comment:
Would be great if we can show this warning comes from content control, since there's same warning log format from hw_codec.c

Reply:
Changed to 
```suggestion
			LOG_WRN("Unexpected/unhandled content control event: %d", event);
```
Also changed to one in hw_codec.c to:
```C
LOG_WRN("Unexpected/unhandled volume event: %d", event);
```